aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -1385,7 +1385,7 @@ Tensor from_file(std::string filename, c10::optional<bool> shared, c10::optional
         c10::StorageImpl::use_byte_size_t(),
         size_bytes,
         THMapAllocator::makeDataPtr(
-            filename.c_str(), flags, size_bytes, nullptr),
+            filename.data(), flags, size_bytes, nullptr),
```

Comment:
This is potentially unsafe, `string_view` isn't necessarily null terminated.

Reply:
Good point. It's probably best if I just change this back to a `std::string`. Loading a tensor from a file is going to take way longer than a string copy anyway